docs/source/portfolio_optimisation/hierarchical_risk_parity.rst

```diff
@@ -39,6 +80,41 @@ Implementation
         .. automethod:: __init__
 
 
+.. note::
+    |h4| Using Custom Input |h4_|
+    We provide great flexibility to the users in terms of the input data - they can either pass their own pre-calculated input
+    matrices/dataframes or leave it to us to calculate them. A quick reference on common input parameters which you will encounter
+    throughout the portfolio optimisation module:
+        * :py:mod:`asset_prices`: Dataframe/matrix of historical raw asset prices **indexed by date**.
+        * :py:mod:`asset_returns`: Dataframe/matrix of historical asset returns. This will be a :math:`TxN` matrix where :math:`T` is the time-series and :math:`N` refers to the number of assets in the portfolio.
+        * :py:mod:`expected_asset_returns`: List of expected returns per asset i.e. the mean of historical asset returns. This refers to the parameter :math:`\mu` used in portfolio optimisation literature. For a portfolio of 5 assets, ``expected_asset_returns = [0.45, 0.56, 0.89, 1.34, 2.4]``.
+        * :py:mod:`covariance_matrix`: The covariance matrix of asset returns.
+
+
+.. tip::
+    |h4| Using Custom Distance Matrix |h4_|
+    The hierarchical clustering step in the algorithm uses a distance matrix to calculate the clusters and form the hierarchical
+    tree. By default, we use the distance matrix mentioned in the original paper,
```

Comment:
Original paper is included only in the docstring of the `HierarchicalRiskParity` class. Perhaps also add it as a Tip?

Reply:
Good idea